islandora_compound_object.module

```diff
@@ -268,17 +268,67 @@ function islandora_compound_object_access(AbstractObject $object) {
 }
 
 /**
- * Replaces parent object thumbnail with child's thumbnail.
+ * Implements hook_compound_cmodel_islandora_derivative().
  */
-function islandora_compound_object_update_parent_thumbnail($parent) {
+function islandora_compound_object_compoundCModel_islandora_derivative(AbstractObject $object = NULL, $ds_modified_params = array()) {
+  $module_path = drupal_get_path('module', 'islandora_solution_pack_compound');
```

Comment:
This will not work. The module is actually 'islandora_compound_object', not 'islandora_solution_pack_compound'. Will need to be `$module_path = drupal_get_path('module', 'islandora_compound_object');`